File: PYTHON/PRAK501-2210817220029-SitiAinurRahmawati.py
def MaxBilangan(a, b, c, d) :
    if a>b and a>c and a>d :
        return a
    elif b>c and b>d and b>a :
        return b
    elif c>b and c>d and c>a :
        return c
    elif d>b and d>c and d>a :
        return d

a, b, c, d = map(int, input().split())
hasil = MaxBilangan(a, b, c, d)
print(hasil)

File: C/PRAK501-2210817220029-SitiAinurRahmawati.c
#include <stdio.h>

//Fungsi MaxBilangan//
int hasil;
int MaxBilangan (int, int, int, int);
int MaxBilangan(int a, int b, int c, int d) {
    if (a>b && a>c && a>d) {
        hasil=a;}
    else if (b>c && b>d && b>a) {
        hasil=b;}
    else if (c>b && c>d && c>a) {
        hasil=c;}
    else if (d>b && d>c && d>a) {
        hasil=d;}
}

int main() {
    int a, b, c, d;
    scanf("%d %d %d %d", &a, &b, &c, &d);
    int hasil = MaxBilangan(a, b, c, d);
    printf("%d", hasil);
    return 0;
}